db/schema.rb

```diff
@@ -226,8 +226,11 @@
     t.bigint "document_id", null: false
     t.datetime "created_at", precision: nil, null: false
     t.datetime "updated_at", precision: nil, null: false
+    t.bigint "user_id"
```

Comment:
The schema looks to have been changed with more than just `scheduled_publication`. Likely some cross over with a later commit

Reply:
resolved, thank u!